src/shared/api/services/appletsService.ts

```diff
@@ -76,7 +80,10 @@ type AppletDetailsRequest = {
   appletId: string;
 };
 
-export type AppletDetailsResponse = SuccessfulResponse<AppletDetailsDto>;
+export type AppletDetailsResponse = {
+  result: AppletDetailsDto;
+  respondentMeta: AppletRespondentMetaDto;
```

Comment:
What if there is no nickname for the respondent? respondentMeta would be null or nickname would be null or empty string?